### PR TITLE
feat(taxjar) :added support for Line item for taxjar

### PR DESCRIPTION
**Pull request by vishdha**
TODO:(Mandatory)
- [x] Development.

...

<h3>Description (Mandatory):</h3>
Added Line items to show item description on taxjar.

Screenshots/GIFs (Mandatory)
image

### CODE REVIEW

File: erpnext/erpnext_integrations/taxjar_integration.py [hunk 1]
```diff
@@ -341,27 +340,25 @@ def sanitize_error_response(response):
 
 	return response
 
-
-def get_line_items(doc):
-	if not doc.items:
-		return
-
-	discout_percent = 0.0
+def get_doc_discount(doc):
```
Comment (neilLasrado) [inline erpnext/erpnext_integrations/taxjar_integration.py:343]: you have not considered shipping here?

**Review by neilLasrado — CHANGES_REQUESTED** [review]

File: erpnext/erpnext_integrations/taxjar_integration.py (hunk 1 above)
Comment (neilLasrado) [reply erpnext/erpnext_integrations/taxjar_integration.py:343]: @vishdha please check this

**Comment by neilLasrado** [comment]
@vishdha what's going on with this PR?

**Comment by vishdha** [comment]
closing this new pull request added https://github.com/Bloomstack/erpnext/pull/1831

**Pull request closed without merging**